### DISCUSSION TITLE
How to set latex-workshop.intellisense.commandsJSON.replace correctly？

### DISCUSSION THREAD

**Discussion by EVISTIX1** [James-Yu/LaTeX-Workshop#3646] [Q&A]
Translation from @James-Yu 

I'd configure auto-completion of LaTeX with VSCode. I got a tutorial from zhihu.com saying to add 
```
    "\\int_{}^{}": {
        "snippet": "\\int_{$1}^{$2}$3\\,\\mathrm{d}",
        "detail": "定积分"
      },
```
to file `C:\Users\用户名\.vscode\extensions\james-yu.latex-workshop-版本号\data\commands.json`

to get the intellisense. However, due to LW updates, `command.json` is overwritten.

So I re-read the LW documentation and discovered a `latex-workshop.intellisense.commandsJSON.replace` config to change and remove auto-completion items.

I followed the official instruction but failed. Can't find other tutorials so I came here.

---
我英文不好，不知道该如何用英文提问，在谷歌搜索多次也无果。这个项目是中国人开发的，我拿中文提问应该可以吧？
问题如下：
我想在配置VScode中配置LaTeX的代码自动补全，在知乎上搜到的教程是在

`C:\Users\用户名\.vscode\extensions\james-yu.latex-workshop-版本号\data\commands.json`

内添加如

`    "\\int_{}^{}": {
        "snippet": "\\int_{$1}^{$2}$3\\,\\mathrm{d}",
        "detail": "定积分"
      },`

的代码即可在VScode内完成代码补全。但是貌似是因为latex-workshop的更新，command.json的内容又被覆盖掉了。
因此我重新阅读latex-workshop的官方文档，发现要在latex-workshop.intellisense.commandsJSON.replace中修改和删除自动补全条目，但是我按照官方文档的示例进行操作没有成功，网上也没找到类似的教程,遂来提问。

**Comment by James-Yu** [comment]
`latex-workshop.intellisense.commandsJSON.replace` is for **modify** and **remove** existing items, not including new ones. If you'd add new ones, you may choose to make them vscode snippets.

---

`latex-workshop.intellisense.commandsJSON.replace` 是用来**修改**和**删除**的，所以**不能添加**新条目。你可以把你的命令做成一个VSCode的snippet。

请尽量提供英文版本供其他人阅读，即使是用翻译软件/网站输出的。大家的问题和回答可能被其他人看到，而他们不一定能读懂中文。
